DirectProgramming/C++SYCL_FPGA/Tutorials/Features/IPAuthoring_interfaces_overview/naive/README.md

```diff
@@ -0,0 +1,184 @@
+# Naive Design
+This example demonstrate a naive implementation of a simple vector addition kernel. 
+
+This design will show interfaces that is invoked by default by the Intel icpx compiler.
+
```

Comment:
https://intel-my.sharepoint.com/personal/paul_white_intel_com/Documents/Documentation/HLS%20Docs/kernel-top-levels.vsdx?web=1

there should be a picture in here in the 'Naive' tab you can use:
![image](https://github.com/KevinUTAT/oneAPI-samples_wenkaixu_fork/assets/86071037/1e8f3080-d58a-437d-8df1-c14ed1cb9205)